sabnzbd/cfg.py

```diff
@@ -613,6 +613,23 @@ def validate_default_if_empty(root: str, value: str, default: str) -> Tuple[None
 pushbullet_prio_queue_done = OptionBool("pushbullet", "pushbullet_prio_queue_done", False)
 pushbullet_prio_other = OptionBool("pushbullet", "pushbullet_prio_other", True)
 
+# [apprise]
+apprise_enable = OptionBool("apprise", "apprise_enable")
+apprise_cats = OptionList("apprise", "apprise_cats", ["*"])
+apprise_urls = OptionStr("apprise", "apprise_urls")
+apprise_prio_startup = OptionBool("apprise", "apprise_prio_startup", False)
+apprise_prio_download = OptionBool("apprise", "apprise_prio_download", False)
+apprise_prio_pause_resume = OptionBool("apprise", "apprise_prio_pause_resume", False)
+apprise_prio_pp = OptionBool("apprise", "apprise_prio_pp", False)
+apprise_prio_complete = OptionBool("apprise", "apprise_prio_complete", True)
+apprise_prio_failed = OptionBool("apprise", "apprise_prio_failed", True)
+apprise_prio_disk_full = OptionBool("apprise", "apprise_prio_disk_full", True)
+apprise_prio_new_login = OptionBool("apprise", "apprise_prio_new_login", False)
+apprise_prio_warning = OptionBool("apprise", "apprise_prio_warning", False)
+apprise_prio_error = OptionBool("apprise", "apprise_prio_error", False)
+apprise_prio_queue_done = OptionBool("apprise", "apprise_prio_queue_done", False)
+apprise_prio_other = OptionBool("apprise", "apprise_prio_other", True)
```

Comment:
Shouldn't we also have priorities? Since we have direct integration of Apprise now, then we could specify it, right?

Reply:
I took this to a slightly different level so i could accommodate the power of apprise.  If i only focused on PushOver Priorities, you'd only be addressing 1 of the 100+ services Apprise can adapt to.  See general description in PR for updated screenshot.